x-pack/plugin/profiler/src/main/java/org/elasticsearch/xpack/profiler/TransportGetProfilingAction.java

```diff
@@ -126,6 +126,8 @@ private void searchEventGroupByStackTrace(
     ) {
         long start = System.nanoTime();
         GetProfilingResponseBuilder responseBuilder = new GetProfilingResponseBuilder();
+        int exp = eventsIndex.getExponent();
+        responseBuilder.setSamplingRate(eventsIndex.getSampleRate());
```

Comment:
Here the naming is slightly inconsistent. We use once the term "sampling rate" and once the term "sample rate". Could you please rename the getter to `getSamplingRate()` so the terminology is consistent?

Reply:
Done :)

Reply:
(Although I have renamed the setter to `setSampleRate` instead).